### PR TITLE
Toggle Sidebar

### PR DESCRIPTION
**Pull request by psarkar2018**

### CODE REVIEW

File: client/src/components/GameArea.js [hunk 1]
```diff
@@ -290,7 +305,7 @@ class GameArea extends Component {
             </Container>
           </Col>
           {/* Sidebar for exile,graveyard,hand,library  */}
-          <Col xs="2" className="sidebar-col p-0 m-0">
+          <Col xs="2" className="sidebar-col p-0 m-0" style={displaySidebar}>
```
Comment (acmorales) [inline client/src/components/GameArea.js]: Are you using `displaySidebar` to show/hide the sidebar? If so, you could use a [conditional render](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) for it.
```
{this.state.isToggleSidebarOn && (
  <Col xs="2" className="sidebar-col p-0 m-0">
    ...
  </Col>
)}
```

Comment (psarkar2018) [reply client/src/components/GameArea.js]: Nice suggestion, Andrew! Will do!

**Comment by psarkar2018** [comment]
I made the changes! Would it be okay to merge this (I know Jake hasn't had a chance to look at this yet, but the changes are pretty small, so maybe its okay merge)?

**Comment by acmorales** [comment]
Looks good! Sorry it took me a bit to get back to this but you're good to merge 👍

**Pull request merged**